ihp-sg13g2/libs.ref/sg13g2_stdcell/lef/sg13g2_tech.lef

```diff
@@ -127,7 +127,7 @@ LAYER Metal2
   WIDTH 0.39        0.21    0.24    0.24
   WIDTH 10.0        0.21    0.24    0.60 ;
   MINIMUMCUT 2 WIDTH 1.4 ;
-  AREA      0.144 ;
+  AREA      0.18 ;
```

Comment:
This is an issue with OpenROAD and should be fixed there. I would suggest to add a comment mentioning why you add this hack and also add the original value.

Reply:
I added a comment to the PR and the commit message. As soon as this gets fixed upstream I'm going to revert this hack. We should both be notified once the issue in ORFS is closed. I don't think a comment is necessary as the old value is saved in the git history.